### PR TITLE
Add @KeyWeeUsr as maintainer

### LINKED ISSUE #9: Add kivy/pyjnius team member(s) as maintainer
@hadim I think that it would be useful to have representation from kivy/pyjnius team on the maintainer list. Thanks to your great work it seems that this recipe is pretty stable now and maintenance effort would be small for additional maintainers. If you agree I will start an appropriate issue on the pyjnius repo, asking for (a member of) the team to represent kivy/pyjnius on the maintainer list.

### PR DESCRIPTION
**Pull request by hanslovsky**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)

Adding @KeyWeeUsr to the maintainers to have a member of the kivy/pyjnius team on the maintainers list.

Closes #9 
Closes kivy/pyjnius#380

@KeyWeeUsr please confirm in this PR that you would like to be added as a maintainer to this repository.

Thanks

### CODE REVIEW

**Comment by KeyWeeUsr** [comment]
Yes, thanks! I'd like to be a maintainer for the pyjnius-feedstock repo. :)

**Comment by hanslovsky** [comment]
All checks pass, will merge once you confirm @hadim

**Review by hadim — APPROVED** [review]

**Pull request merged**
